calculator/math_parser.py

```diff
@@ -20,6 +20,7 @@
 
 from collections import deque
 import re
+import sys

```

Comment:
where is `sys` import used?

Reply:
Ohh, upps I used this for exit the program. I changed my mind. I remove it.